tracker/forms.py

```diff
@@ -44,25 +46,45 @@ def __init__(self, *args, request=None, **kwargs):
         # Remove user parameter - not needed for now
         kwargs.pop('user', None)
         super().__init__(*args, **kwargs)
-        # Make mileage field read-only in the form
-        self.fields['mileage'].widget.attrs['readonly'] = True
-        self.fields['mileage'].help_text = 'Calculated automatically from default location'
+        
+        # If creating a new game, hide mileage field
+        # If editing an existing game, show mileage as editable
```

Comment:
ok good find, would pk be None? then I would do: `if self.instance.ok is not None:`